release_notes_generator/model/issue_record.py

```
@@ -0,0 +1,292 @@
+from typing import Optional, Any, TYPE_CHECKING
+
+from github.Commit import Commit
+from github.Issue import Issue
+from github.PullRequest import PullRequest
+
+from release_notes_generator.action_inputs import ActionInputs
+from release_notes_generator.model.record import Record
+
+# Use TYPE_CHECKING for imports needed only for type hints
+if TYPE_CHECKING:
+    from release_notes_generator.model.pull_request_record import PullRequestRecord
```

Ruff (0.12.2)

`release_notes_generator.model.pull_request_record.PullRequestRecord` imported but unused

Remove unused import: `release_notes_generator.model.pull_request_record.PullRequestRecord`

(F401)